### PR TITLE
Add minimal config subsystem

### PR DESCRIPTION
**Pull request by emersion**

### CODE REVIEW

File: include/sway/config.h [hunk 1]
```diff
@@ -0,0 +1,405 @@
+#ifndef _SWAY_CONFIG_H
+#define _SWAY_CONFIG_H
+
+#define PID_WORKSPACE_TIMEOUT 60
+
+#include <libinput.h>
+#include <stdint.h>
+#include <wlc/geometry.h>
+#include <wlc/wlc.h>
```
Comment (SirCmpwn) [inline include/sway/config.h]: :hankey:

File: include/sway/config.h [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+	int left_handed;
+	int middle_emulation;
+	int natural_scroll;
+	float pointer_accel;
+	int scroll_method;
+	int send_events;
+	int tap;
+
+	bool capturable;
+	struct wlc_geometry region;
```
Comment (SirCmpwn) [inline include/sway/config.h]: :hankey:

**Comment by SirCmpwn** [comment]
Looks good, just get rid of the lingering wlc issues. That's why the bulid failed, too.

**Comment by emersion** [comment]
Ow, I didn't even noticed that. I'll pay more attention to includes in the future.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**